app/views/admin/invoices/show.html.erb

```diff
@@ -0,0 +1,52 @@
+<div class="col-lg-4 rounded" style="border: 2px red; padding: 15px">
+  <div class="card " style="background-color: rgb(191, 202, 209);min-width: 480px">
+    <div class="card-header" style="background-color: rgb(134, 153, 165)">
+      <h3 class="card-title">Invoice # <%= @invoice.id %></h3>
+      <hr
+      class="float-left"
+      style="
+        color: rgb(105, 0, 0);
+        border: 2px solid rgb(112, 0, 0);
+        width: 50%;
+      "
+    />
+      <h4>Status: <%= @invoice.status %></h4>
+      <h4>Ordered: <%= @invoice.formatted_time %></h4>
+      <!-- <h4 style="padding-top: 8px; color: rgb(191, 202, 209);" class="card-title"><%= @invoice.id %> Status: <%= @invoice.status%></h4> -->
+      <h4>Customer: <%= @invoice.customer.first_name + ' ' + @invoice.customer.last_name %></h4>
```

Comment:
You could consider adding an Invoice instance method to return the customer's last name instead of chaining methods.

Reply:
Model method maybe?